app/tile/table-skeleton.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import styled, {keyframes} from "styled-components"
+
+const loading = keyframes`
+  0% { background-position: -500px 0; }
+  100% { background-position: 500px 0; }
+`
+
+const BG = styled.div`
+  animation: fadein 200ms;
+  padding: 12px;
+`
+
+const Row = styled.div`
+  height: 22px;
+  width: 100%;
+  border-radius: 1px;
+  background: linear-gradient(
+    to right,
+    var(--table-stripe-bg) 10%,
+    var(--hover-light-bg) 50%,
+    var(--table-stripe-bg) 90%
+  );
+  animation: ${loading} 1s linear infinite;
+  background-size: 1000px 22px;
+  &:nth-child(even) {
+    background: none;
+  }
+  position: relative;
+`
+
+export default function TableSkeleton() {
+  return (
+    <BG>
+      {Array(10)
```

Comment:
I was gonna suggest to make the number of rows be a prop, with a default of 10 so we could easily adjust based on amount of records from a query with `... | head n`, but I see below that we don't actually know how many records we will get until search is complete and then this component is replace.

So for now, is it the case that we have a constraint to keep our summary queries to 10 results or risk seeing a change in size to the tiles on completion?

Reply:
The tiles will not change size every. They have a fixed width/height. If a table gets too large, the overflow is hidden.

I did add head 10 for all the queries on the board. It seems like a good constraint for now. But yes, I do think that the skeleton should have a prop for the number of rows. I'll add it in a later PR most likely.